packages/es-components/src/components/controls/dropdown/Dropdown.js

```diff
@@ -26,9 +34,20 @@ function Dropdown({
   onChange,
   onBlur,
   validationState,
+  additionalHelpContent,
   theme,
   ...rest
 }) {
+  const helpId = additionalHelpContent ? `${name}-help` : undefined;
+  const additionalHelp = additionalHelpContent && (
+    <AdditionalHelpContent
+      id={helpId}
+      className="dropdown__help"
```

Comment:
`es-dropdown__help` to be consistent

Reply:
For the additional help content all of the class names simply reference the component, I modeled it off of `textbox__help` but we can change it to `es-textbox__help` and update it across dropdown, radiogroup and checkbox.

Reply:
Yes please!